drivers/comparator/qm_comparator.c

```diff
@@ -79,6 +81,9 @@ int qm_ac_set_config(const qm_ac_config_t *const config)
 	QM_SCSS_CMP->cmp_ref_pol = config->polarity;
 	QM_SCSS_CMP->cmp_pwr = config->power;
 
+	/* Provide some delay after power Change to Comparator*/
```

Comment:
does this affect Quark SE C1000 as well?
If not, we would need #if guards
How can this be reproduced?

Thanks